src/modules/devices/pulser/MockNeutralAtomDevice.py

```diff
@@ -0,0 +1,84 @@
+#  Copyright 2021 The QUARK Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import TypedDict
+
+import pulser
+from pulser.devices import MockDevice
+from pulser_simulation import QutipBackend
+
+from modules.devices.pulser.Pulser import Pulser
+from modules.Core import Core
+
+
+class MockNeutralAtomDevice(Pulser):
+    """
+    Class for using the local mock Pulser simulator for neutral atom devices
+    """
+
+    def __init__(self):
+        """
+        Constructor method
+        """
+        super().__init__(device_name="mock neutral atom device")
+        self.device = MockDevice
+        self.backend = QutipBackend
+        self.submodule_options = []
+
+    def get_parameter_options(self) -> dict:
+        """
+        Returns the configurable settings for this application
+        """
+        return {
+            "doppler_noise": {
```

Comment:
When I ran benchmark runs with noise, I got the error "ValueError: 'doppler_noise' is not a valid noise type. Valid noise types: doppler, amplitude, SPAM, dephasing, depolarizing, eff_noise." So, I changed all four names of the optional noises and now it works fine.

Reply:
noise parameter names changed by removing "_noise" suffix.